src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -96,9 +96,11 @@ class outputStream;
   f(conc_class_unload_purge_ec,                     "    Exception Caches")            \
   f(conc_strong_roots,                              "Concurrent Strong Roots")         \
   SHENANDOAH_PAR_PHASE_DO(conc_strong_roots_,       "  CSR: ", f)                      \
-  f(conc_rendezvous_roots,                          "Rendezvous")                      \
   f(conc_evac,                                      "Concurrent Evacuation")           \
                                                                                        \
+  f(disable_weakroots_gross,                        "Disable Weak Roots (G)")          \
+  f(disable_weakroots,                              "Disable Weak Roots (N)")          \
```

Comment:
Should start with "Pause". And should probably be something generic, like "Pause Final Roots"? This probably percolates to the names of everything else...